src/lib/Scenes/Artwork/Components/ImageCarousel/ImageWithLoadingState.tsx

```diff
@@ -1,9 +1,6 @@
-import { color } from "@artsy/palette"
```

Comment:
I removed all of the custom loading stuff from this file because it was looking far too busy after the placeholder change.